ompi/mpi/fortran/use-mpi-f08/mpi-f-interfaces-bind.h

```diff
@@ -655,10 +655,10 @@ subroutine ompi_type_create_subarray_f(ndims,array_of_sizes, &
    INTEGER, INTENT(OUT) :: ierror
 end subroutine ompi_type_create_subarray_f
 
-subroutine ompi_type_dup_f(type,newtype,ierror) &
+subroutine ompi_type_dup_f(oldtype,newtype,ierror) &
```

Comment:
FWIW, these `[p]ompi_*_f` interfaces don't matter -- they're the behind-the-scenes interfaces that are not user-facing.  It doesn't hurt to update them as well, but it also doesn't matter.